coap-gateway/test/test.go

```diff
@@ -16,7 +15,7 @@ import (
 	"go.uber.org/zap/zapcore"
 )
 
-func MakeConfig(t *testing.T) service.Config {
+func MakeConfig(t require.TestingT) service.Config {
```

Comment:
What's the difference between using this or `*testing.T`? This way the `testing` package is not imported?

Reply:
`*testing.T` is a struct that Im not able construct in mains of `test/services/cmd` / `test/virtual-device/cmd`.  `require.TestingT` is `interface{}` so in am able to satisfy interface there so I can use test code in these test binaries.